src/pres/loader/PGNLoader.js

```diff
@@ -16,14 +16,18 @@ export default class PGNLoader extends React.Component {
     constructor(props) {
         super(props)
         let selectedSite = new URLSearchParams(window.location.search).get("source")
+        let playerName = new URLSearchParams(window.location.search).get("playerName")
+        let color = new URLSearchParams(window.location.search).get("color") // 'white' | 'black'
+        // 'all' | 'rated' | 'casual' (corresponds to Filters.toggleRated)
+        let ratedMode = new URLSearchParams(window.location.search).get("ratedMode")
 
         this.state = {
-            playerName: '',
+            playerName: playerName?playerName:'',
```

Comment:
I'm imitating style of `selectedSite?selectedSite:''`, but maybe should add some spaces and/or use `||` (or `??`, depending)

(Also in a few other places)